src/form/Calendar/CalendarDays/CalendarDays.tsx

```diff
@@ -174,6 +241,13 @@ export const CalendarDays: FC<CalendarDaysProps> = ({
           opacity: { duration: 0.2, type: animated ? 'tween' : false }
         }}
       >
+        <div className={css.week}>
+          {dayLabels.map((label, index) => (
+            <div key={`day-label-${index}`} className={css.dayLabel}>
+              {label}
+            </div>
+          ))}
+        </div>
```

Comment:
this looks like an added feature - we need to make it so that it a prop to turn on and default it to off otherwise anyone who upgrades to the latest will see unexpected changes